core/server/src/main/webapp/browse-body.jsp

```diff
@@ -16,6 +16,33 @@
 <%@ page import="static java.net.URLEncoder.encode" %>
 
 <jsp:include page="header-scripts.jsp" />
+
+<script>
+    var currentDir = "<%= request.getAttribute("currentPath").toString()%>";
+    function getInputPath() {
+        var path = $.trim($("#pathInput").val());
+        return path;
+    }
+
+    function changeDir() {
+        var path = getInputPath();
+        path = encodeURI(path);
+        window.location.href="./browse?path="+path;
+    }
+
+    $(document).ready(function () {
+        $("#pathInput").val(currentDir);
+        $("#goBtn").click(changeDir);
+        $("#pathInput").keydown(function (e) {
+            if(e.keyCode==13){
+                changeDir();
+            }
+        });
+        if(base != "./browse") {
```

Comment:
The designer of the browse page and browseLog page are the same one, i think.
The http requst of browse page and browseLog page are both use browse.jsp to handle, but as you know, the log page dosen't need a nav bar.
I must thank you for your suggestion.